tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -93,8 +93,8 @@ def test_create_new_study_id_multiple_studies():
     storage = create_test_storage()
     session = storage.scoped_session()
 
-    study_id_1 = storage.create_new_study_id()
-    study_id_2 = storage.create_new_study_id()
+    study_id_1 = storage.create_new_study()
```

Comment:
`id` suffix still remains in the method name.

Reply:
Thank you for pointing it out.
I fixed the names: https://github.com/pfnet/optuna/pull/506/commits/a21678d945aca2914bd3de0e646a39a76375b9e0